src/components/Dropdown/Dropdown.tsx

```diff
@@ -93,6 +94,10 @@ const DropdownMenuContent = styled(GenericMenuPanel)`
   min-width: ${({ theme }) => theme.click.genericMenu.item.size.minWidth};
   flex-direction: column;
   z-index: 1;
+  overflow-y: scroll;
+  max-height: calc(
+    (var(--radix-${({ $type }) => $type}-content-available-height) - 100px)
```

Comment:
Out of curiosity, what does `100px` represent?

Reply:
@hoorayimhelping this subtracts 100px from the available height so the bottom of the dropdown doesn't get too close to the bottom of the screen per Daniel's request